src/cmap/commands.ts

```diff
@@ -856,7 +855,7 @@ export class BinMsg {
       if (payloadType === 0) {
         const bsonSize = this.data.readUInt32LE(this.index);
         const bin = this.data.slice(this.index, this.index + bsonSize);
-        this.documents.push(raw ? bin : BSON.deserialize(bin, bsonOptions));
+        this.documents.push(raw ? bin : deserialize(bin, bsonOptions));
```

Comment:
Is this change (`BSON.des` vs just `des`) something to make sinon work? I think that's fine let's just make it the same for all usages and serialize usages.

Reply:
Yup, this was just to sinon work.  But importing from `src/bson` instead of `'bson'` fixed the issue that I was seeing earlier, and I can revert it.